modules/react-components/src/layouts/list.tsx

```diff
@@ -56,6 +56,10 @@ export interface ListLayoutPropsInterface extends PaginationProps, IdentifiableC
      * Disables Right action panel component.
      */
     disableRightActionPanel?: boolean;
+    /**
+     * Check if the list is loading
+     */
+    isLoading?: boolean;
```

Comment:
Shall we make `isLoading` prop to a mandatory one? So people won't miss to add it.

Reply:
Will be adding this in the future as an improvement.